UnityProject/Assets/Scripts/Managers/UpdateManager/UpdateManager.cs

```diff
@@ -244,6 +247,7 @@ private void RemoveCallbackInternal(CallbackCollection collection, Action callba
 
 	private void Update()
 	{
+		CashedDeltaTime = Time.deltaTime;
```

Comment:
Codacy found an issue: [Make the enclosing instance method 'static' or remove this set on the 'static' field.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6392909)